platform/ios/src/MGLMapView.mm

```diff
@@ -1167,7 +1167,8 @@ - (void)touchesBegan:(__unused NS_SET_OF(UITouch *) *)touches withEvent:(__unuse
 }
 
 - (void)notifyGestureDidBegin {
-    [self notifyMapChange:mbgl::MapChangeRegionWillChange];
+    bool animated = false;
```

Comment:
I'm a bit confused - I've seen `bool` usage when the code is a glue between C++ and Objective-C, and `BOOL` on Objective-C code along this source file - should we open an issue then to fix this for all remaining `bool`s?